sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -179,6 +180,14 @@ def assert_all_properties(self, properties, value):
         assert properties.can_write == value
         assert properties.can_list == value
 
+    def create_fully_qualified_reference(self, registry, repository, digest):
```

Comment:
not sure what this is being used for, but it seems similar to the `fully_qualified_reference` property in the `_models.py` file. can you call that instead of writing a separate method for this/is this needed? or is this something completely different and unrelated?

Reply:
This recreates the `fully_qualifed_reference` property as it should look, it's just a helper method to construct the string and assert they are the same (or one is within the other).